build_deploy.sh

```diff
@@ -4,9 +4,27 @@ set -exv
 
 source 'scripts/deploy/build-deploy-common.sh'
 
-IMAGE_NAME="${IMAGE_NAME:-quay.io/cloudservices/compliance-backend}"
-ADDITIONAL_TAGS="latest"
-REQUIRED_REGISTRIES_LOCAL=''
-BUILD_ARGS=("IMAGE_TAG")
+# Check if the current Git branch is 'origin/security-compliance'.
+if [[ $GIT_BRANCH == "origin/security-compliance" ]]; then
+    IMAGE_NAME="${IMAGE_NAME:-quay.io/cloudservices/compliance-backend}"
+    
+    # Generate a tag for the Docker image based on the current date and Git commit short hash.
+    SECURITY_COMPLIANCE_TAG="sc-$(date +%Y%m%d)-$(git rev-parse --short=7 HEAD)"
+    
+    # Set ADDITIONAL_TAGS to the generated security compliance tag.
+    ADDITIONAL_TAGS="$SECURITY_COMPLIANCE_TAG"
 
-build_deploy_main || exit 1
+    REQUIRED_REGISTRIES_LOCAL=''
+    BUILD_ARGS=("IMAGE_TAG")
+
+    build_deploy_main || exit 1
```

Comment:
I believe you can exclude this line out of the if clause, as well as the `BUILD_ARGS` . Essentially I'd keep anything that remains constant outside of the IF clause

Reply:
@Victoremepunto , That is a good call! 

I have updated the PR.